PKGBUILDS/framebufferphone/fbp/PKGBUILD

```
@@ -0,0 +1,28 @@
+# Maintainer: Miles Alan <m@milesalan.com>
+# Contributor (on arch): Hamblingreen <hamblingreen@hotmail.com>
+
+pkgname=fbp
+pkgver=0.4
+pkgrel=0
+pkgdesc="Framebuffer linux phone UI environment, supported best on the Pinephone"
+arch=('aarch64' 'x86_64')
```

```suggestion
arch=('x86_64' 'armv7h' 'aarch64')
```

Even if we don't build ARMv7 yet, it should still be added.